py_mind_memo/persistence.py

```
@@ -34,18 +34,25 @@ def on_save_as(self, event=None):
             data = self.model.save()
-            with open(file_path, "w", encoding="utf-8") as f:
-                json.dump(data, f, ensure_ascii=False, indent=4)
+            self._perform_write_to_file(file_path, data)
             self.current_file_path = file_path
             self.model.is_modified = False
+            return True
         except Exception as e:
             messagebox.showerror("Error", f"Failed to save to {file_path}: {e}")
+            return False
+
+    def _perform_write_to_file(self, file_path, data):
```

Ruff (0.15.7)

[warning] Missing return type annotation for private function `_perform_write_to_file`

Add return type annotation: `None`

(ANN202)